src/main/java/org/dependencytrack/persistence/jdbi/FindingDao.java

```diff
@@ -0,0 +1,560 @@
+                     CASE WHEN "VULNERABILITY"."SEVERITY" = 'UNASSIGNED' 
+                          THEN 0 
+                          WHEN "VULNERABILITY"."SEVERITY" = 'LOW' 
+                          THEN 3 
+                          WHEN "VULNERABILITY"."SEVERITY" = 'MEDIUM' 
+                          THEN 6 
+                          WHEN "VULNERABILITY"."SEVERITY" = 'HIGH' 
+                          THEN 8 
+                          WHEN "VULNERABILITY"."SEVERITY" = 'CRITICAL' 
+                          THEN 10 
+                          ELSE CASE WHEN "VULNERABILITY"."CVSSV3BASESCORE" IS NOT NULL 
+                                    THEN "VULNERABILITY"."CVSSV3BASESCORE" 
+                                    ELSE "VULNERABILITY"."CVSSV2BASESCORE" 
+                               END 
+                     END 
+                     """),
+            @AllowApiOrdering.Column(name = "vulnerability.cvssV3BaseScore", queryName = "\"VULNERABILITY\".\"CVSSV3BASESCORE\""),
+            @AllowApiOrdering.Column(name = "vulnerability.cvssV2BaseScore", queryName = "\"VULNERABILITY\".\"CVSSV2BASESCORE\""),
+            @AllowApiOrdering.Column(name = "vulnerability.published", queryName = "\"VULNERABILITY\".\"PUBLISHED\""),
+            @AllowApiOrdering.Column(name = "attribution.analyzerIdentity", queryName = "\"FINDINGATTRIBUTION\".\"ANALYZERIDENTITY\""),
+            @AllowApiOrdering.Column(name = "component.projectName", queryName = "concat(\"PROJECT\".\"NAME\", ' ', \"PROJECT\".\"VERSION\")"),
+            @AllowApiOrdering.Column(name = "component.name", queryName = "\"COMPONENT\".\"NAME\""),
+            @AllowApiOrdering.Column(name = "component.version", queryName = "\"COMPONENT\".\"VERSION\""),
+            @AllowApiOrdering.Column(name = "analysis.state", queryName = "\"ANALYSIS\".\"STATE\""),
+            @AllowApiOrdering.Column(name = "analysis.isSuppressed", queryName = "\"ANALYSIS\".\"SUPPRESSED\""),
+            @AllowApiOrdering.Column(name = "attribution.attributedOn", queryName = "\"FINDINGATTRIBUTION\".\"ATTRIBUTED_ON\"")
+    })
+    @AllowUnusedBindings
+    @RegisterConstructorMapper(FindingRow.class)
+    List<FindingRow> getAllFindings(@Define String queryFilter,
+                                 @Define boolean activeFilter,
+                                 @Define boolean suppressedFilter);
+
+    /**
+     * Returns a List of all Finding objects filtered by ACL and other optional filters.
+     * @param filters        determines the filters to apply on the list of Finding objects
+     * @param showSuppressed determines if suppressed vulnerabilities should be included or not
+     * @param showInactive   determines if inactive projects should be included or not
+     * @return a List of Finding objects
+     */
+    default PaginatedResult getAllFindings(final AlpineRequest alpineRequest, final Map<String, String> filters, final boolean showSuppressed, final boolean showInactive) {
```

Comment:
Filtering by severity seems to be broken:

![image](https://github.com/user-attachments/assets/f27b0b74-d693-42d3-b064-bd9776769f05)

```
2025-03-19 11:38:09,740 ERROR [GlobalExceptionHandler] Uncaught internal server error [requestId=279585c1-9c70-40a5-b23a-b47ba89b3c4e]
org.jdbi.v3.core.statement.UnableToCreateStatementException: Missing named parameter 'severity0' 
...
```

Reply:
Same problem with other filters like analysis status, vendor response, etc.

```
org.jdbi.v3.core.statement.UnableToCreateStatementException: Missing named parameter 'analysisStatus0'
...
org.jdbi.v3.core.statement.UnableToCreateStatementException: Missing named parameter 'vendorResponse0'
```

I think the parameters are not bound correctly.